apps/address-book/package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "solium --dir ./contracts",
     "precommit": "lint-staged",
     "prepublishOnly": "truffle compile --all",
-    "publish:http": "npm run build:script && aragon apm publish major --http localhost:1111 --http-served-from ./dist",
+    "publish:http": "npm run build:script && aragon apm publish major --http localhost:1111 --http-served-from ./dist --propagate-content false --skip-confirmation true",
```

Comment:
This is not related to using experimental versions of the wrapper, but to upgrading the `@aragon/cli` that is not needed at all for the scope of this task. I also spent lot of time extensively researching the update strategy, so I would like to ask you to revert these changes and wait until I present my solution to use the new cli.

To clarify again: We don't need to use the newest @aragon/cli versions for the scope of this PR

Just use the `--client-version` option to test Camino client version (that should be also reflected on the title/description of the PR if you decide to do so.

The client is a different thing from the wrapper and the api and `@aragon/cli` is different from `@aragon/aragon` aka **the client™️** and definitely unrelated to `@aragon/aragon.js` (the monorepo that contains the wrapper and api that the title mentions)

https://hack.aragon.org/docs/cli-main-commands#aragon-run

Reply:
Another alternative is using `npm link` if you need to test alternative repos locally, we've been doing this last year and it does not represent any difficulty, I think that is a better solution than merging these kind of changes (using special repo versions) into the dev branch, at least just for the sake of using a different `cli` that is not needed here

Reply:
We need a new version of the CLI in order to add the `--client-repo` option to the `aragon run` command, so that we can use our fork of the client.

> Another alternative is using `npm link` if you need to test alternative repos locally, we've been doing this during the whole year and it does not represent any difficulty

"Does not represent _any_ difficulty"? Maybe I'm doing it wrong; let's have a call so you can set me straight. I don't think it's a maintainable solution to _require_ all members of our team to do this (with specific branch of the client, linked to a specific branch of aragon.js) just to have baseline-functional apps during development. Also, when linking, I think I observed some script.js code gets minified, which results in much less actionable stack traces. This slows me down while working on web3 integration for Projects, and I don't think I can afford any slow-downs there. But again, maybe I configured something incorrectly in my environment, and there's a zero-hassle way to accomplish this.

I agree that needing to upgrade the CLI is unfortunate; I would love to avoid it if we can. Using `npm link` did not seem like a workable solution to me. Let's have a call and figure it out.